feature/sessions/README.md

```diff
@@ -76,24 +76,24 @@ install(Sessions) {
 
 **Notes about security:**
 
-* Serving session without transforms allows people to see in the clear the contents of the session and to modify it
-* Serving session with an Authentication transform allows people to see the contents, but prevents modifying it as long
-  as you keep your secret hash key safe and use a secure algorithm. Also allows to use old session strings to go back
+* Serving a session without transform allows people to see the contents of the session clearly and to modify it
+* Serving a session with an Authentication transform people can see the contents, but prevents them from modifying it as long
+  as you keep your secret hash key safe and use a secure algorithm. It is also possible to use old session strings to go back
   to a previous state.
-* Serving session with an Encrypt transform prevents from actual determining the contents and preventing modification,
-  but still is vulnerable to going back to previous states.
+* Serving a session with an Encrypt transform prevents people from determining the actual contents and modification it,
+  but it is still vulnerable when going back to previous states.
```

Comment:
What I intended to say here is that a soneone can just take a previous session and to use it again later.

For example:

```
Server says: available_points=10_VERIFICATION_9812812BCD122 <-- A person trying to exploit this, gets this string
Client says: available_points=10_VERIFICATION_9812812BCD122

Server says: available_points=9_VERIFICATION_121356736CDVD
Client says: available_points=10_VERIFICATION_9812812BCD122 <--- A person trying to exploit this, uses an old value (previous state) that works without modification.
```

So it is not "when going back", but that the user forces to use an old value.

Reply:
Ah ok, sorry!
Then:
but it is still vulnerable to exploitation and being returned to previous states.